timber/src/main/java/timber/log/Timber.kt

```diff
@@ -271,6 +270,17 @@ class Timber private constructor() {
     }
   }
 
+  /**
+   * A [Tree] for debug builds.
+   * Automatically shows a Hyperlink to the calling Class and Linenumber in the Logs.
+   * Allows quick lookup of the caller source just by clicking on the Hyperlink in the Log.
+   * @param showMethodName Whether or not to show the method name as well
+   */
+  class HyperlinkedDebugTree(private val showMethodName: Boolean = true) : DebugTree() {
+    override fun createStackElementTag(element: StackTraceElement) =
+            with(element) { "($fileName:$lineNumber) ${if (showMethodName) " $methodName()" else ""}" }
```

Comment:
Wanted to keep it short, but might have turned out too short.
I think it's still easily readable though so I left it like this instead of spreading it over more lines.